lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1270,6 +1273,110 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     break;
   }
 
+  case Kinded::Kind::RowwiseQuantizedFullyConnectedInstKind: {
+    auto *RWQFC = cast<RowwiseQuantizedFullyConnectedInst>(I);
+    // Since we can't get the variable from a glow::Value directly,
+    // we need to traverse the var list and find the one matching the given
+    // Value.
+    Tensor scalesT;
+    auto *F_ = getIRFunction();
+    for (auto &v : F_->getGraph()->getParent()->getVars()) {
```

Comment:
would this be a useful utility for other cases?

Reply:
So far, I don't think it is a common case to get the variable from a glow::Value like this.  Actually, we can add the scales and offsets as "Member" instead of "addInput" or "addOperand" , and they will be passed as ArrayRef as how we pass the strides vector in Convolution. But it requires extra work on LLVMIRGen::emit*** when passing the param to the kernels. But I think in the current way, the code itself looks more clean :)